tests/components/cloud/test_google_config.py

```diff
@@ -105,30 +107,27 @@ async def test_google_entity_registry_sync(hass, mock_cloud_login, cloud_prefs):
     await config.async_connect_agent_user("mock-user-id")
 
     with patch.object(
-        config, "async_sync_entities", side_effect=mock_coro
+        config, "async_schedule_google_sync_all", side_effect=mock_coro
     ) as mock_sync, patch.object(ga_helpers, "SYNC_DELAY", 0):
+        # Created entity
         hass.bus.async_fire(
             EVENT_ENTITY_REGISTRY_UPDATED,
             {"action": "create", "entity_id": "light.kitchen"},
         )
         await hass.async_block_till_done()
 
-    assert len(mock_sync.mock_calls) == 1
+        assert len(mock_sync.mock_calls) == 1
 
-    with patch.object(
-        config, "async_sync_entities", side_effect=mock_coro
-    ) as mock_sync, patch.object(ga_helpers, "SYNC_DELAY", 0):
+        # Removed entity
         hass.bus.async_fire(
             EVENT_ENTITY_REGISTRY_UPDATED,
             {"action": "remove", "entity_id": "light.kitchen"},
         )
         await hass.async_block_till_done()
 
-    assert len(mock_sync.mock_calls) == 1
+        assert len(mock_sync.mock_calls) == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5412406)